FrontEnd/staticwebapp.config.json

```diff
@@ -53,7 +53,9 @@
     }
   },
   "globalHeaders": {
-    "content-security-policy": "default-src https: 'unsafe-eval' 'unsafe-inline'; object-src 'none'"
+    "content-security-policy": "default-src https: 'unsafe-eval' 'unsafe-inline'; object-src 'none'",
+    "Access-Control-Allow-Origin": "*",
```

Comment:
Is `"Access-Control-Allow-Origin": "*"` necessary? This would allow other websites to make requests to our API

Reply:
It is not! I checked if CORS were at fault and forgot to clean it up.